scenarios/12_Leaving_Raedwood.cfg

```diff
@@ -262,6 +280,43 @@ Move Ruvio to signpost to ask him for advice."}
         {INTERROGATION12}
     [/event]
 
+    [event]
+        name=last breath
+        [filter]
+            id=Quivre O Raedbor
+        [/filter]
+        [filter_second]
+            id=Gawen Hagarthen
+        [/filter_second]
+        {MSGW_Ruvio _"Gawen! I thought I told you not to waste your time with that one!"}
+        {MSGW_Gawen _"I know, I was just curious to see if he had anything to say."}
+        {MSG_unit _"Well, I don't! So there!"}
+        [kill]
+            id=Quivre O Raedbor
+            # (no animation here, because it's ok to leave it ambiguous whether he died or just left here)
+        [/kill]
+        {MSGW_Ruvio _"See? What did I tell you?"}
```

Comment:
See above for comments on Ruvio/Gawen's speech patterns.

Reply:
I made another stab at it in 6f04548 but I'm not sure if that's an improvement...